platforms/documentation/docs/src/docs/userguide/authoring-builds/other/custom_gradle_types.adoc

```diff
@@ -338,3 +339,114 @@ A link:{javadocPath}/org/gradle/api/DomainObjectSet.html[DomainObjectSet] simply
 Compared to `NamedDomainObjectContainer`, a `DomainObjectSet` doesn't manage the objects in the collection. They need to be created and added manually.
 
 You can create an instance using the link:{javadocPath}/org/gradle/api/model/ObjectFactory.html#domainObjectSet-java.lang.Class-[ObjectFactory.domainObjectSet()] method.
+
+[[custom_dependencies_blocks]]
+== Plugin provided `dependencies` block
```

Comment:
```suggestion
== Customizing dependency declarations
```

The name of the block isn't the important part.  I think this documentation should get after from emphasizing it like this.  Readers might get the impression the name matters.  Might even be worth updating the example to use `myDependencies` to really hammer this point.

Reply:
This heading makes it seem like you're customizing dependency declarations anywhere, but the point is that as a plugin author, you're providing a place to put dependency declarations related to your plugin.

I'm also not sure it's really important for us to show the full flexibility (you don't have to name it `dependencies`)

Reply:
I tried incorporating some of this into the new title